doc-content/enterprise-only/release-notes/rn-720-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,56 @@
+[id='rn-720-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.2.0
+
+{PRODUCT} 7.2.0 provides increased stability and fixed issues listed in this section.
+
+== Installation
+* The {PRODUCT} installer fails to parse the `jboss-cli.xml` file with {EAP} 7.2 and the installation fails [https://issues.jboss.org/browse/RHPAM-1658[RHPAM-1658]]
+
+== {CENTRAL}
+* It is not possible to edit an existing deployment descriptor configuration [https://issues.jboss.org/browse/RHPAM-1318[RHPAM-1318]]
+* Process Instances pages that have a vertical scroll bar flicker after changing window width [https://issues.jboss.org/browse/RHPAM-1475[RHPAM-1475]]
+* Opening the legacy case modeler throws a JSON exception [https://issues.jboss.org/browse/RHPAM-1431[RHPAM-1431]]
+* Incorrect correlation key is shown in the {PRODUCT} Task list [https://issues.jboss.org/browse/RHPAM-1656[RHPAM-1656]]
+* You cannot open a task form from the Task inbox in {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1565[RHPAM-1565]]
+* The data set editor displays the `supports expression reference $null` message as a hint [https://issues.jboss.org/browse/RHPAM-1581[RHPAM-1581]]
+* The *Asset* search button is not positioned properly [https://issues.jboss.org/browse/RHPAM-1538[RHPAM-1538]]
+* The input fields in the form for uploading files into {Central} are not aligned [https://issues.jboss.org/browse/RHDM-4[RHDM-4]]
+ifdef::PAM[]
+* Provide configuration options for paginated views [https://issues.jboss.org/browse/RHPAM-1339[RHPAM-1339]]
+* Provide CORS configuration support for {KIE_SERVER} in {PRODUCT} [https://issues.jboss.org/browse/RHPAM-1434[RHPAM-1434]]
+endif::PAM[]
+ifdef::DM[]
+* Provide configuration options for paginated views [https://issues.jboss.org/browse/RHDM-1339[RHDM-1339]]
+* In a test scenario, you cannot select a rule defined in xlsx in EXPECTATION settings [https://issues.jboss.org/browse/RHDM-728[RHDM-728]]
+* Provide CORS configuration support for {KIE_SERVER} in {PRODUCT} [https://issues.jboss.org/browse/RHDM-520[RHDM-520]]
+
+== Decision engine
+* ThreadSafeTrackableTimeJobFactoryManager for default [https://issues.jboss.org/browse/RHDM-759[RHDM-759]]
```

Comment:
`ThreadSafeTrackableTimeJobFactoryManager` set as default in `SessionConfigurationImpl.java`